CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2.15.0-rc.0 / 2019-12-18
+
+* [CHANGE] Discovery: Removed `prometheus_sd_kubernetes_cache_*` metrics. #6393
+* [FEATURE] API: Added new endpoint for exposing per metric metadata `/metadata`. #6420 #6442
+* [FEATURE] Remote write: Added config option to name remote target which is added as another label dimension to `prometheus_remote_storage_*` metrics. #6043
+* [FEATURE] Metrics format: Added examplar support to OpenMetrics parser. #6292
```

Comment:
This is not user visible

Reply:
Hm, but suers can expose exemplars in metric page now, right?  (: even though we don't use them really.

Reply:
They could do that for some time, but we're still basically dropping them on the floor.

Reply:
Fair, we then did not change anything useful for users